.lando.yml

```diff
@@ -9,48 +9,21 @@ config:
   database: mariadb
   xdebug: false
 
+env_file:
+  - config/.env
+  - .env
```

Comment:
Why do we need to have both these in here?

Reply:
The `config/.env` is the base one. Then `.env` is `.gitignored` for you to be able to add overrides. For example, in mine I have just one environment variable to allow running from src:
```bash
# Location of WordPress core directory to run from
WP_CORE_DIR=/app/public/core-dev/src/
```